engine/admin/logging/awslogs.md

```diff
@@ -63,6 +63,16 @@ specified, the container ID is used as the log stream.
 > at a time.  Using the same log stream for multiple containers concurrently
 > can cause reduced logging performance.
 
+### tag
+
+Specify `tag` as an alternative to the `awslogs-stream` option. `tag` interprets template markup (e.g., `{{.ID}}`, `{{.FullID}}` or `{{.Name}}` `docker.{{.ID}}`). 
+See the [tag option documentation](log_tags.md) for details on all supported template substitutions.
+
+When both `awslogs-stream` and `tag` are specified, the value supplied for `awslogs-stream` will override the template specified with `tag`.
+
+By default, the `{{.FullID}}` template will be used to tag log messages.
```

Comment:
I think this is covered in the `awslogs-stream` section, though it might bear repeating here.

> If not specified, the container ID is used as the log stream.